SECURITY.md

```diff
@@ -9,7 +9,7 @@ please check
 [the latest version](https://github.com/flatpak/xdg-desktop-portal-gtk/blob/master/SECURITY.md).
 
 | Version   | Supported          | Status
-| --------- | ------------------ | -------------------------------------------------------------- |
-| 1.15.x    | :hammer:           | Development branch, releases may include non-security changes  |
-| 1.14.x    | :white_check_mark: | Stable branch, recommended for use in distributions            |
-| <= 1.13.x | :x:                | Older branches, no longer supported                            |
+| --------- | ------------------ | ----------------------------------------------------------- |
+| 1.15.x    | :white_check_mark: | Current stable branch, recommended for use in distributions |
+| 1.14.x    | :white_check_mark: | Old stable branch, will received only security fixes        |
```

Comment:
```suggestion
| 1.14.x    | :white_check_mark: | Old stable branch, will receive only security fixes         |
```

Reply:
We could follow the GNOME pattern of alpha/beta/rc, but to be honest I think we can get away without the complication. After all, xdp-gtk is an implementation of interfaces; if those change, we are going to bump up the minor version anyway, and I'd expect xdg-desktop-portal to be the one component that has a full development cycle.